src/transforms/interval.js

```diff
@@ -9,9 +9,19 @@ function maybeInterval(interval) {
   if (typeof interval === "number") {
     const n = interval;
     // Note: this offset doesn’t support the optional step argument for simplicity.
-    interval = {floor: d => n * Math.floor(d / n), offset: d => d + n};
+    interval = {
+      floor: d => n * Math.floor(d / n),
+      mid: d => n * (Math.floor(d / n) + 0.5),
+      offset: d => d + n
+    };
```

Comment:
Also, this isn’t sufficient to test for a numeric interval, since you could provide your own implementation of the floor and offset functions, and we’d have to call those to determine whether or not your interval is numeric or temporal. This code assumes every implementation is a temporal interval. (See isTimeInterval, as I mentioned.)